config/database.yml.hudson

```diff
@@ -2,6 +2,7 @@ default: &default
   adapter: postgresql
   encoding: unicode
   user: panoptes
+  username: panoptes
```

Comment:
Yes. This is a stupid thing in rails (though from what I can tell it's fixed in 5.0). It dumps the sql format using the postgres command line tools, and sets variables for the database connection through the PG* environment vars. For some reason it sets `PGUSER` from a `username` field instead of the `user` field which it uses for connections otherwise.